strands_navigation_msgs/msg/Edge.msg

```diff
@@ -6,9 +6,8 @@ string map_2d
 bool use_default_recovery
 string[] monitor_recovery_pairs_names
 strands_navigation_msgs/DynClassLoaderDef[] monitor_states
-strands_navigation_msgs/DynClassLoaderDef[] recovery_state_machines
 bool use_default_nav_recovery
```

Comment:
why did you remove this one? its needed to define recovery behaviours for bumper, magnetic strip, etc